### PR TITLE
chore: continue using the `defaultOpen` name

### PR DESCRIPTION
**Pull request by patrickdevivo**
but address the underlying console error

### CODE REVIEW

File: packages/blocks/src/components/organisms/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -116,9 +116,10 @@ const SidebarItem: React.FC<
   compact = true,
   subNav,
   level,
+  defaultOpen,
```
Comment (patrickdevivo) [inline packages/blocks/src/components/organisms/Sidebar/Sidebar.tsx:119]: @german-mergestat by adding it here, it should address the message you were seeing in the console before

Comment (german-mergestat) [reply packages/blocks/src/components/organisms/Sidebar/Sidebar.tsx:119]: ok let me release a new version and test it from `mergestat`.

Comment (german-mergestat) [reply packages/blocks/src/components/organisms/Sidebar/Sidebar.tsx:119]: yep it worked... I did the change in this [PR](https://github.com/mergestat/blocks/pull/144/files). I think approving that PR will close this one.

**Comment by patrickdevivo** [comment]
Closing in favor of https://github.com/mergestat/blocks/pull/144

**Pull request closed without merging**